dev/preview/infrastructure/harvester/vm.tf

```diff
@@ -75,11 +77,19 @@ resource "kubernetes_secret" "cloudinit" {
   }
 }
 
+data "kubernetes_secret" "harvester-k3s-dockerhub-pull-account" {
```

Comment:
Reading this from a k8s secret, as it seems that I (and I imagine other people as well) don't have permissions to read from secrets manager in `core-dev`

Reply:
Sounds like a fine solution for now - we can always streamline later